packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -140,12 +140,12 @@ export class VaAccordion {
 
   private accordionsOpened(method='every') {
     // Track user clicks on va-accordion-item within an array to compare if all values are true or false
-    let accordionItems = [];
-    getSlottedNodes(this.el, 'va-accordion-item').forEach(item => {
-      accordionItems.push((item as Element).getAttribute('open'));
-    });
-    const allOpen = currentValue => currentValue === 'true';
-    const allClosed = currentValue => currentValue === 'false';
+    const accordionItems = [...this.el.children]
+      .filter((el) => el.tagName.toLowerCase() === 'va-accordion-item')
+      .map((el) => el.open);
```

Comment:
The change here is to get the accordion items to determine their open state without relying on the DOM being hydrated yet with the `va-accordion-item` element. This update also adds the items to the array using a non-mutating method (spread operator, filter, map).